Composer/packages/ui-plugins/orchestrator/src/index.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { PluginConfig, ShellApi } from '@bfc/extension-client';
+import { SDKKinds, DialogInfo } from '@bfc/shared';
+import formatMessage from 'format-message';
+
+const config: PluginConfig = {
+  uiSchema: {
+    [SDKKinds.OrchestratorRecognizer]: {
+      recognizer: {
+        disabled: (shellAPI: ShellApi) => shellAPI.featureFlags('ORCHESTRATOR'),
+        displayName: () => formatMessage('Orchestrator Recognizer'),
```

Comment:
this is what I meant, having displayName as string vs. as function

displayName: formatMessage('SomeText'); // incorrect usage of format-message
vs.
 displayName: () => formatMessage('SomeText'); // correct usage